var/spack/repos/builtin/packages/circuit-build/package.py

```diff
@@ -14,6 +14,7 @@ class CircuitBuild(PythonPackage):
 
     version('develop', branch='master')
     version('3.1.1', tag='circuit-build-v3.1.1')
+    version('3.1.3', tag='circuit-build-v3.1.3.dev0')
```

Comment:
Is this going to turn into a non dev tag?

Reply:
Yes, I'm waiting for approval on this tag. This is just a test. Btw how to test PR's module now? I tried the usual:
```
  $ module purge
  $ unset MODULEPATH
  $ source /gpfs/bbp.cscs.ch/apps/hpc/jenkins/pulls/xxx/config/modules.sh
```
but received
```
-bash-4.2$ module purge
-bash-4.2$ unset MODULEPATH
-bash-4.2$ source /gpfs/bbp.cscs.ch/apps/hpc/jenkins/pulls/1029/config/modules.sh
-bash-4.2$ module load circuit-build
ModuleCmd_Load.c(213):ERROR:105: Unable to locate a modulefile for 'circuit-build'
```

Reply:
`module load unstable circuit-build` should work, I hope!

Reply:
Yep, I tried but the old version `3.1.1` was loaded.
```
-bash-4.2$ module load unstable
-bash-4.2$ module load circuit-build/3.1.3
ModuleCmd_Load.c(213):ERROR:105: Unable to locate a modulefile for 'circuit-build/3.1.3'
```

Reply:
I think there's a version pin in `deploy/environments/applications.yaml`. If you remove the version from `circuit-build` there, Jenkins should build the newest version automatically.

Reply:
thank you!